README.md

```diff
@@ -84,6 +84,15 @@ and put it in `~/Anki/addons`.
 
 [addons21]: https://apps.ankiweb.net/docs/addons.html#_add_on_folders
 
+### AnkiDroid
+
+Advanced -> Custom sync server
+
+Sync url: `http://your.server:27701`
+Media sync url: `http://your.server:27701/msync`
+
+Make sure to not use https, no trailing slashes and include the port. You'll be asked to login again, the email is the username (no full email necessary).
```

Comment:
HTTPS works just fine. I have HTTPS set up and it works with zero issues. And the login definitely doesn't have to be your email. My server doesn't know anything about emails.

Reply:
No it doesn't. It requires a reverse proxy being set up: https://github.com/tsudoko/anki-sync-server/issues/17 -- Login: Please re-read what I wrote, that's what I am pointing out :)

Reply:
You shouldn't be using this without a reverse proxy unless you are on a secure private network (and even then...). Saying "Make sure not to use https" suggests there will be issues if you try to use it behind a reverse proxy. You also don't have to include the port if it is the standard one for the protocol (80 for http, 443 for https), which you may or may not choose to use on your proxy.

I'm not saying that the information you are trying to get across might not be useful, just that the way you put it there makes things worse, IMHO.